src/main/java/org/example/bookstoreapp/service/BookService.java

```diff
@@ -0,0 +1,11 @@
+package org.example.bookstoreapp.service;
+
+import java.util.List;
+import org.example.bookstoreapp.model.Book;
+
+public interface BookService {
+
+    Book save(Book book);
```

Comment:
There is no need to add this empty line here (and in any other classes)

```suggestion
public interface BookService {
    Book save(Book book);
```

Reply:
done